RefreshTests.GameServer/RefreshTests.GameServer.csproj

```diff
@@ -49,6 +49,11 @@
 
     </ItemGroup>
 
+    <ItemGroup>
+      <None Remove="Resources\1x1.png" />
+      <EmbeddedResource Include="Resources\1x1.png" />
```

Comment:
Why two?

Reply:
idk rider does that

Reply:
Gotcha. I think only the second is necessary